pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -318,8 +318,20 @@ func (c *Cluster) configureMgrModules() error {
 
 		if module.Enabled {
 			if module.Name == balancerModuleName {
+				force := false
+				// Get current "client" features
+				features, err := client.CheckFeatures(c.context, c.Namespace)
+				if err != nil {
+					return errors.Wrap(err, "failed to check current cluster features")
+				}
+				// If a client is on "jewel" we need to force the activation otherwise it will fail
+				for _, client := range features.Client {
+					if client.Release == "jewel" {
```

Comment:
Is `jewel` the only client that we care about? Might there be clients in the list even older than jewel? But like my question above, maybe we should just always force it.